spec/recipes/windows_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe 'pantry::default' do
+  let(:chef_run) do
+    ChefSpec::SoloRunner.new(platform: 'windows', version: '2012').
+      converge(described_recipe)
+  end
+
+  it 'includes chocolatey recipe' do
+    expect(chef_run).to include_recipe('chocolatey')
```

Comment:
again, implementation detail - i don't think this should be tested specifically.

This like the other include_recipe statements in the os x specific recipe above are implementation details. There's little value here because they're unconditionally included, and then we're just testing that chef works. The include recipe for the platforms in general (in the default recipe) has value because we're ensuring that we get the correct thing based on the platform.